ios/RNCAsyncStorage.m

```diff
@@ -447,6 +556,16 @@ - (NSDictionary *)_writeEntry:(NSArray<NSString *> *)entry changedManifest:(BOOL
 
 RCT_EXPORT_METHOD(getAllKeys:(RCTResponseSenderBlock)callback)
 {
+  if (self.delegate != nil) {
+    [self.delegate allKeys:^(NSArray<id<NSObject>> *keys) {
+      callback(@[(id)kCFNull, keys]);
+    }];
+
+    if (![self _passthroughDelegate]) {
```

Comment:
Is this opposite the meaning? I think if `passThrough` is `YES`, we should pass through built-in storage.

Reply:
If the delegate is a passthrough, it means that it wants to receive the event and also let the signal pass through it. So if it isn't, it will swallow the event, or return here.

Reply:
@tido64 Seems I misunderstand the `passthrough`.